options/auto_retry_options.go

```diff
@@ -14,4 +14,6 @@ var RETRYABLE_ERRORS = []string{
 	"(?s).*Error installing provider.*TLS handshake timeout.*",
 	"(?s).*Error configuring the backend.*TLS handshake timeout.*",
 	"(?s).*Error installing provider.*tcp.*timeout.*",
+	"(?s).*Error installing provider.*tcp.*timeout.*",
```

Comment:
Duplicated lines 16 and 17.

Reply:
Good catch! https://github.com/gruntwork-io/terragrunt/pull/754